package.json

```diff
@@ -94,7 +94,7 @@
       "esm"
     ]
   },
-  "packageManager": "pnpm@7.14.2",
+  "packageManager": "pnpm@8.3.1",
```

Comment:
Looks like pnpm was updated on main, this ensures that corepack users (like me 🙂) will use 8.x.